include/ck/tensor_operation/gpu/device/impl/codegen_device_grouped_conv_fwd_multiple_abd_xdl_cshuffle.hpp

```diff
@@ -727,6 +727,200 @@ struct CodegenDeviceGroupedConvFwdMultipleABD_Xdl_CShuffle
         ck::Array<index_t, NDimSpatial> input_right_pads_;
     };
 
+    static bool IsSupportedArgument(const Argument& arg)
```

Comment:
Should you add this IsSupported check in some place of code? I mean in some test/example before convolution call.

Reply:
I have added it as part of the template